content/docs/command-reference/import.md

```diff
@@ -104,15 +104,18 @@ To actually [version the data](/doc/start/data-and-model-versioning), `git add`
   > [Importing and updating fixed revisions](#example-importing-and-updating-fixed-revisions)
   > example below).
 
-- `--no-exec` - create the import `.dvc` file but don't download `url` (assumes
+- `--no-exec` - create the import `.dvc` file without accessing `url` (assumes
   that the data source is valid). This is useful if you need to define the
   project imports quickly, and import the data later (use `dvc update` to finish
   the operation(s)).
 
-- `--no-download` - create the import `.dvc` with data checksums but without
-  downloading the associated data. This is useful if you need track changes in
-  remote data but do not (yet) need to download data to the local workspace.
-  Data can be later downloaded using `dvc pull`.
+- `--no-download` - create the import `.dvc` file including the
+  [version](https://git-scm.com/docs/revisions) for the external dependency but
+  without downloading the associated data. This is useful if you need track
```

Comment:
Term "external dependency" is out of context in this page. The description has this phrase though: "[.dvc](https://dvc.org/doc/user-guide/project-structure/dvc-files) files support references to data in an external DVC repo..." so maybe something like:

```suggestion
- `--no-download` - create the import `.dvc` file including the
  reference to the external data source but
  without downloading the associated data. This is useful if you need track
```

Or if we want to keep the link to Git revision concept (unclear it's helpful here IMO though), maybe something like

```suggestion
- `--no-download` - create the import `.dvc` file including the source data
  [version](https://git-scm.com/docs/revisions) but
  without downloading the associated data. This is useful if you need track
```